kibble/config/config_test.go

```diff
@@ -41,3 +39,9 @@ func TestLoadLanguagesConfigWithObjects(t *testing.T) {
 	assert.Equal(t, cfg.Languages["it"].Code, "it_IT")
 	assert.Equal(t, cfg.Languages["it"].Name, "Italian")
 }
+
+func TestLoadConfig(t *testing.T) {
+	sitePath = "../sample_site/kibble.json"
+	cfg := LoadConfig(false, "xxx", false)
```

Comment:
Just for people wondering what `xxx` is...

```suggestion
	cfg := LoadConfig(false, "api key", false)
```